airflow/providers/openlineage/plugins/adapter.py

```diff
@@ -375,18 +377,29 @@ def dag_started(
             # This part cannot be wrapped to deduplicate code, otherwise the method cannot be pickled in multiprocessing.
             self.log.warning("Failed to emit DAG started event: \n %s", traceback.format_exc())
 
-    def dag_success(self, dag_run: DagRun, msg: str):
+    def dag_success(
+        self,
+        dag_id: str,
+        run_id: str,
+        end_date: datetime,
+        execution_date: datetime,
+        state: DagRunState,
```

Comment:
I'd change that to `dag_run_state` everywhere - we are passing tasks here as well so let's avoid any possible confusions.

Reply:
Fixed.